src/services/subscription/subscription-router.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+import { SubscriptionValidator } from "./subscription-schema";
+import { Database } from "../../database";
+
+const subscriptionRouter = Router();
+
+// Create a new subscription
+subscriptionRouter.post("/subscribe", async (req, res, next) => {
+    try {
+        const subscriptionData = SubscriptionValidator.parse(req.body);
+        await Database.SUBSCRIPTION.create(subscriptionData);
+        return res.status(StatusCodes.CREATED).json(subscriptionData);
```

Comment:
Instead of creating, add to array like this:

```typescript
Database.SUBSCRIPTIONS.findOneAndUpdate(
    { mailingList: data.mailingList }, 
    { $push: { subscriptions: data.email } },
    {upsert: true}
);
```